### PR TITLE
[issues forms] new Forms design using YAML schema markdown

### PR DESCRIPTION
**Pull request by arrowgent**
example results:
https://github.com/arrowgent/dosbox-staging/issues

**test yourself by starting an issue**

files changed:
https://github.com/arrowgent/dosbox-staging/tree/master/.github/ISSUE_TEMPLATE

issue conversation:
https://github.com/dosbox-staging/dosbox-staging/issues/1233

recent Beta release:
https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/

converting from template markdown to YAML:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

_im new to YAML_
i think `Version` can be reworked to automatically pull from Releases,  if possible somehow to automatically add new lines here from your workflows?
so you dont have to always update the version in bug_report.yaml

### CODE REVIEW

**Comment by arrowgent** [comment]
other ideas:

add:

```
  - type: checkboxes
    id: checked
    attributes:
      label: Have you checked that no other similar issue already exists?
      description: Searched issues before creating report?
      options:
        - label: I have searched and not found similar issues.
          required: true
```

**Comment by arrowgent** [comment]
adding URLS:

contributing guidelines and code of conduct. 

https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md
https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md

to final line

`      description: By submitting this issue, you agree to follow our Code of Conduct`

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      description: How can we reproduce this?
+      placeholder: "Explain how to reproduce"
+    validations:
+      required: false
+  - type: textarea
+    id: config_file
+    attributes:
+      label: Share the config file(s) you've been using to run the program.
+      description: (dosbox-staging.conf) Please avoid pasting the full config, use attachments or links
+      placeholder: "example: texture_renderer = opengl"
```
Comment (Dobby233Liu) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:91]:
```suggestion
      placeholder: "example: texture_renderer = opengl"
      render: ini # syntax highlighting
```

Comment (arrowgent) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:106]: render:
not supported by input:
only supported in textarea:

Comment (arrowgent) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:106]: https://github.com/arrowgent/dosbox-staging/issues/12

example

Comment (Dobby233Liu) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:106]: isn't this textarea?

Comment (arrowgent) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:106]: sorry, wasnt sure how to integrate it with my changes & commits.

you can of course use my fork 

https://github.com/arrowgent/dosbox-staging/tree/issueforms

**Comment by arrowgent** [comment]
note to remove the big amount of commits, i can delete and create the 3 files again in a new PR
minimizing the spam of me editing

**Comment by arrowgent** [comment]
yes, so remove editing spam i deleted and will recreate PR

**Pull request closed without merging**